lib/src/common/variables.js

```diff
@@ -26,8 +29,9 @@ export const globalTokens = {
   hal_purple_s_38: "#5f249f",
   hal_purple_d_30: "#4b1c7d",
   hal_purple_d_20: "#321353",
-  hal_purple_d_70: "#9a6bb2",
-  color_purple_600: "#7d2fd0",
+  hal_purple_d_70: "#9A6BB2",
+  color_purple_600: "#7D2FD0",
+  color_purple_700: "#5F249F",
   color_purple_300: "#cbacec",
```

Comment:
`#5f249f` already exists in `hal_purple_s_38`.

Reply:
done